src/components/AuthContainer.js

```diff
@@ -14,8 +15,11 @@ import TermsOfService from 'src/pages/TermsOfService'
 const AuthContainer = ({ children }) => {
   const { name, public: isPublic } = useRoute()
   const { isSignedIn, registrationStatus, acceptedTos, profile } = useStore(authStore)
+  const auth = useAuth()
   const authspinner = () => h(centeredSpinner, { style: { position: 'fixed' } })
 
+  useOnMount(() => authStore.update(_.set(['authContext'], auth)))
+
```

Comment:
A companion `AuthStoreSetter` component could be used alongside the suggestion below.
```javascript
export const AuthStoreSetter = () => {
  const auth = useAuth()
  useOnMount(() => authStore.update(_.set(['authContext'], auth)))
  return null
}
```